packages/fxa-payments-server/src/components/NewUserEmailForm/index.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { NewUserEmailForm } from './index';
+
+storiesOf('components/NewUserEmailForm', module).add('default', () => (
+  <div style={{ display: 'flex' }}>
+    <NewUserEmailForm
+      onBlurInitial={(evt) => {
+        console.log('onBlurInitial: ', evt);
+      }}
+      onBlurConfirm={(evt) => {
+        console.log('onBlurInitial: ', evt);
+      }}
```

Comment:
I don't see these logging when I blur these `Input` fields in the form. Do they log for you?

I do see a tooltip that says "Sign in with your Firefox Account", but I assume we'll change that once [email validation](https://mozilla-hub.atlassian.net/browse/FXA-3665) and/or [existing account checking](https://mozilla-hub.atlassian.net/browse/FXA-3663) is implemented.

Reply:
This is a placeholder for the email validation calls

Reply:
> This is a placeholder for the email validation calls

I think Bianca meant that when she focus on the input and the blur, she did not see any console.log output.  Even if they are placeholders, they should execute?